internal/pkg/service/cli/cmd/remote/create/table/dialog.go

```diff
@@ -103,14 +100,15 @@ func AskCreateTable(args []string, branchKey keboola.BranchKey, allBuckets []*ke
 		panic(errors.New("unexpected state"))
 	}
 
-	if f.PrimaryKey.IsSet() {
+	if f.PrimaryKey.Value != "" {
 		opts.CreateTableRequest.PrimaryKeyNames = strings.Split(strings.TrimSpace(f.PrimaryKey.Value), ",")
 	} else {
```

Comment:
`else` -> `else if !f.PrimaryKey.IsSet()`

If the user enters `--primary-key ""`, it means, that no primary key should be created.
- So the user has explicitly specified their will and the dialog box should be skipped.